src/Aquifer.Public.API/Endpoints/Resources/Collections/Get/Response.cs

```diff
@@ -0,0 +1,26 @@
+using System.Text.Json.Serialization;
+using Aquifer.Common.Utilities;
+using Aquifer.Data.Entities;
+
+namespace Aquifer.Public.API.Endpoints.Resources.Collections.Get;
+
+public sealed class Response
+{
+    public required string Code { get; init; }
+    public required string DisplayName { get; init; }
+    public required string ShortName { get; init; }
+    public required ResourceType ResourceType { get; init; }
+
+    [JsonConverter(typeof(JsonUtilities.RawJsonConverter))]
+    public required string? LicenseInfo { get; init; }
```

Comment:
I only exposed `LicenseInfo` from here in the GET route, not in the List route.  Is that acceptable?

Reply:
I think it should be okay.